cli/src/cmd/forge/script/build.rs

```diff
@@ -148,9 +145,11 @@ impl ScriptArgs {
                         extra.target_id = Some(id.clone());
                     }
                 } else {
-                    let split: Vec<&str> = extra.target_fname.split(':').collect();
-                    let path = std::path::Path::new(split[0]);
-                    let name = split[1];
+                    let (path, name) = extra
+                        .target_fname
+                        .rsplit_once(':')
+                        .expect("The target specifier is malformed.");
```

Comment:
Turns out the bug was staring us in the face all along - on Windows paths obviously contain `:` so the path we compared with in this branch was literally just the drive letter on Windows lmao

Reply:
omg